charts/cloudflared/templates/service.yaml

```diff
@@ -19,7 +19,7 @@ spec:
     {{- if $val.enabled }}
     - name: {{ $key | quote }}
       port: {{ $val.port }}
-      targetPort: {{ $key | quote }}
+      targetPort: 5053
```

Comment:
`targetPort` references the named port inside the deployment ("dns" with the default values) - it should be left as is. 🙂

Reply:
I know, but I find it confusing, as we define a port called dns three lines before. But that is another port definition and not the same one that is being used in the deployment specs.
So, for clarity I changed it.

Reply:
Yes, the name definition three lines above is only for the service and the `targetPort` value references a port in the deployment. To keep it consistent the ports should be named equally in my opinion. Also a hardcoded port does not work here as it would also hardcode the metrics port.